Source: Julian Ospald <hasufell@posteo.de>
Upstream: no
Reason: Fix installation destination

diff -ru mygui-MyGUI3.2.2.orig/CMake/ConfigureBuild.cmake mygui-MyGUI3.2.2/CMake/ConfigureBuild.cmake
--- mygui-MyGUI3.2.2.orig/CMake/ConfigureBuild.cmake	2015-01-26 10:32:35.000000000 -0500
+++ mygui-MyGUI3.2.2/CMake/ConfigureBuild.cmake	2015-03-14 01:09:40.743621643 -0400
@@ -4,6 +4,8 @@
 # also prepare package files for pkg-config and CMake.
 #######################################################################
 
+include(GNUInstallDirs)
+
 # should we build static libs?
 if (MYGUI_STATIC)
   set(MYGUI_LIB_TYPE STATIC)
@@ -47,7 +49,7 @@
   else ()
     configure_file(${MYGUI_TEMPLATES_DIR}/MYGUI.pc.in ${MYGUI_BINARY_DIR}/pkgconfig/MYGUI${MYGUI_LIB_SUFFIX}.pc @ONLY)
   endif ()
-  install(FILES ${MYGUI_BINARY_DIR}/pkgconfig/MYGUI${MYGUI_LIB_SUFFIX}.pc DESTINATION lib/pkgconfig)
+  install(FILES ${MYGUI_BINARY_DIR}/pkgconfig/MYGUI${MYGUI_LIB_SUFFIX}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 
   # configure additional packages
   
